.travis.yml

```diff
@@ -0,0 +1,6 @@
+osx_image: xcode7.3
+language: node_js
+node_js:
+  - "4.4.x"
+script:
+  - npm install
```

Comment:
It's useful for me to see what builds break on what operating system all in one place.